### DISCUSSION TITLE
waline 设置true不显示，还有哪里需要设置呢

### DISCUSSION THREAD

**Discussion by TYblog** [YunYouJun/hexo-theme-starry#286] [Q&A 疑难杂症]
![image](https://user-images.githubusercontent.com/35173696/165967548-a4658a80-718b-457d-b553-3bdc628b1494.png)

**Comment by YunYouJun** [comment]
请检查控制台信息。

**Reply by TYblog** [reply]
给您添麻烦。waline中comment:false，控制台没有任何错误。comment:true，无法运行报错Cannot read property 'indexOf' of undefined。![image](https://user-images.githubusercontent.com/35173696/166086455-01649b27-9f0c-4abd-8f0b-2f72f4bb8793.png)

**Comment by YunYouJun** [comment]
请检查是否存在某处图标名称误设置为空。
